proposals/description/fip_76a.ts

```diff
@@ -0,0 +1,41 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_x: ProposalDescription = {
+  title: 'FIP-76a: Add LUSD to FeiRari',
+  commands: [
+    {
+      target: 'fuseAdmin',
+      values: '0',
+      method: 'oracleAdd(address[],address[])',
+      arguments: [['{lusd}'], ['{rariChainlinkPriceOracleV3}']],
+      description: 'Set oracle for LUSD'
+    },
+    {
+      target: 'fuseAdmin',
+      values: '0',
+      method: '_deployMarket(address,address,string,string,address,bytes,uint256,uint256,uint256)',
+      arguments: [
+        '{lusd}', // underlying
+        '{rariPool8DaiIrm}', // IRM
+        'FeiRari LUSD', // Name
+        'fLUSD-8', // Symbol
+        '{rariPool8CTokenImpl}', // impl
+        '0x', // constructor bytes (not used)
+        '200000000000000000', // reserve factor 20%
+        '0', // no admin fee
+        '800000000000000000' // LTV 80%
+      ],
+      description: 'Add LUSD to FeiRari'
+    },
+    {
+      target: 'fuseGuardian',
```

Comment:
Raising the G-UNI DAI/FEI supply cap is seperate functionality to adding LUSD to the FeiRari pool.  I guess because it's non-controversial and a small change, it's fine to combine in the same OA action

Reply:
we usually don't do governance for technical changes. The problem here is that we changed the incentives but the cap is too low in the Fuse pool to welcome all the tokens that once were staked in the TribalChief